src/components/header/modals/ModalNetwork.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <astar-simple-modal :show="isOpen" title="Network" @close="closeModal">
+    <div class="wrapper--modal-network">
+      <div class="wrapper--select-network">
+        <div class="row--separator--account">
+          <div class="border--separator--account" />
+        </div>
+        <fieldset>
+          <ul role="radiogroup">
+            <li v-for="(provider, index) in providerEndpoints" :key="index" class="tw-mb-2">
+              <label
+                :class="[
+                  'class-radio',
+                  selNetwork === index ? 'class-radio-on' : 'class-radio-off',
+                ]"
+              >
+                <input
+                  name="choose_networks"
+                  type="radio"
+                  :checked="selNetwork === index"
+                  class="
+                    ip--network
+                    tw-appearance-none
+                    tw-border-2
+                    tw-border-gray-100
+                    tw-rounded-full
+                    tw-h-4
+                    tw-w-4
+                    tw-mr-3
+                    focus:tw-outline-none
+                    tw-bg-white
+                    checked:tw-border-4
+                  "
+                  @change="selNetwork = index"
+                />
+                <div class="tw-text-left tw-flex-1">
```

Comment:
![Screen Shot 2022-04-04 at 10 22 22 AM](https://user-images.githubusercontent.com/40356749/161503899-6e2bf8ce-39a7-4e03-9465-dfd93c33da9f.png)

The input position seems to be off

Reply:
@sirius651 @hoonsubin 
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/325) PR.